src/ddspdrum/module.py

```diff
@@ -368,16 +368,21 @@ def _npyforward(self, mod_signal: np.array, phase: float = 0.0) -> np.ndarray:
 
```

Comment:
hmmmm so changing numpy functionality that has already been ported to torch (VCO), you should also port these changes to torch.